README.md

```diff
@@ -61,56 +63,65 @@ pip install noteable-origami
 
 ## Getting Started
 
-Get your API token from Noteable within user settings.
-Within user settings, go to the API Token page, and generate a new token.
 
-```python
-from origami.client import NoteableClient
+### API Tokens
 
-token = ''  # Your API token from Noteable
+The Noteable API requires an authentication token. You can manage tokens at the Noteable user settings page.
 
-# Establish a connection to the realtime API
-async with NoteableClient(api_token=token) as client:
-    await client.ping_rtu()
-```
+1. Log in to Noteable (sign up is free)
+2. In the User Settings tab, navigate to `API Tokens` and generate a new token
 
-### Token via Environment Variable
+### Usage
 
-Alternatively you can set the environment variable:
+The example below shows how to create a Notebook, launch a Kernel, add new cells, and execute code. 
 
-```bash
-NOTEABLE_TOKEN=xxxx
-```
+```python
+# Grab a project_id from the Noteable UI, the url will look like: app.noteable.io/p/....
+api_token = '...'
+project_id = '...'
 
-and skip assigning the token:
+# Client for interacting with Noteables REST API
+from origami.clients.api import APIClient
+api_client = APIClient(api_token)
 
-```python
-async with NoteableClient() as client:
-    await client.ping_rtu()
-```
+# Create a new Notebook
+file = await api_client.create_notebook(project_id=project_id, path="Demo.ipynb")
 
-### Custom Domain
+# Start a Kernel
+await api_client.launch_kernel(file.id)
 
-```bash
-NOTEABLE_TOKEN=xxxx
-NOTEABLE_DOMAIN=app.noteable.io
-```
+# Client for Real-time Updates (RTU), used with Notebooks
+rtu_client = await api_client.rtu_client(file.id)
 
-And the client will use that particular domain, for custom deployment location. This value defaults to `app.noteable.io`.
+# Add a new cell
+from origami.models.notebook import CodeCell
+cell = CodeCell(source="print('Hello World')")
+await rtu_client.add_cell(cell)
 
-```python
-async with NoteableClient() as client:
-    await client.ping_rtu()
+# Execute the cell
+queued_execution = await rtu_client.execute_cell(cell.id)
+
+# Wait for the execution to be complete
+cell = await queued_execution
+
+# Grab the output
+output_collection = await api_client.get_output_collection(cell.output_collection_id)
+output_collection.outputs[0].content.raw
+>>> 'Hello World\n'
```

Comment:
If you're going to show inputs and outputs, the input starts with `>>>` while the output is on a blank line.

```
>>> def f():
...   return 3
...
>>> f()
3
```

However, I'd use the `In` and `Out` formatting from IPython:

```
In [1]: from origami.models.notebook import CodeCell
   ...: cell = CodeCell(source="print('Hello World')")
   ...: await rtu_client.add_cell(cell)
```